### PR TITLE
Prettier Applied for good understanding and readability

### PR DESCRIPTION
**Pull request by Harry262530**
### 🛠️ Fixes Issue
Added Prettier Linter for proper documentation

### 👨‍💻 Changes proposed
Indentation and Bugs are taken care 

### ✔️ Check List (Check all the applicable boxes) 

-  My code follows the code style of this project.
-  This PR does not contain plagiarized content.
-  The title of my pull request is a short description of the requested changes.

### 📄 Note to reviewers
I have applied prettier extension for every file that can made it easier for new developers

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/workflows/linting.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Linting
```
Comment (rohanbabbar04) [inline .github/ISSUE_TEMPLATE/workflows/linting.yml:1]: Change it to prettier, same with the filename

**Review by rohanbabbar04 — COMMENTED** [review]
Hi thanks for this it looks great :)
Just some minor changes, won't take much time
-  I would want the linting to work only for the project files, not the other workflow files...
- Also would be good to add a section name `Prettier` in the `Contributing Guidelines`, so that the contributors are aware and can use prettier before making commits.

File: .github/ISSUE_TEMPLATE/workflows/linting.yml (hunk 1 above)
Comment (Harry262530) [reply .github/ISSUE_TEMPLATE/workflows/linting.yml:1]: sure

Comment (Harry262530) [reply .github/ISSUE_TEMPLATE/workflows/linting.yml:1]: can you please label it under gssoc 23

Comment (rohanbabbar04) [reply .github/ISSUE_TEMPLATE/workflows/linting.yml:1]: Once it is ready to merge I will label it...

**Pull request closed without merging**